init.lua

```diff
@@ -82,7 +82,7 @@ local function update(widget, reg, disablecache)
     if c and c.time and c.data and t < c.time+reg.timer and not disablecache then
         return update_value(format_data(c.data))
     elseif reg.wtype then
-        if reg.wtype.async then
+        if reg.wtype().async then
```

Comment:
This will run the function, which is not what we want. In our case widgets are tables, where the `__call` attribute is set so they are also callable.